packages/matplotlib/patches/fix-threading.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/lib/matplotlib/backend_bases.py b/lib/matplotlib/backend_bases.py
+--- a/lib/matplotlib/backend_bases.py
++++ b/lib/matplotlib/backend_bases.py
+@@ -1993,6 +1993,7 @@ class FigureCanvasBase:
+ 
+     def draw(self, *args, **kwargs):
+         """Render the `.Figure`."""
++        pass
+ 
+     def draw_idle(self, *args, **kwargs):
+         """
+diff --git a/lib/matplotlib/backends/backend_agg.py b/lib/matplotlib/backends/backend_agg.py
+--- a/lib/matplotlib/backends/backend_agg.py
++++ b/lib/matplotlib/backends/backend_agg.py
+@@ -22,6 +22,8 @@ TODO:
+ """
+ 
+ try:
++    # force use of dummy_threading
++    raise ImportError()
```

Comment:
What happens if you keep using the threading import? Recently in https://github.com/iodide-project/pyodide/pull/796 I have removed all `dummy_threading` patches, using it for actual threading will lead to deadlocks anyway. While `threading` works for locks etc in Python 3.8

Reply:
I'll try. Thought it was broken in Pyodide.

Reply:
It seems to work! There is a `threading.Timer` used in `font_manager.py` just to delay a warning (it was also broken with dummy threading). I'll remove it.